CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### [0.4.0] - 2021-11-06
+* Add job details endpoint (API v2)
+* Add support for pagination of results
```

Comment:
Maybe should include the details about results now coming as a list rather than a list under resp["items"], since this is the thing that might be breaking for existing users?